lintconf.yaml

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Thanks for your contribution! Wondering why this file is needed 🤔 ?

Reply:
Without this file ct-lint apprenetly throws an error in the latest version, atleast when you try to execute `make helm-lint`. That's jsut a yamllint config in relaxed mode.